server/src/test/java/org/spine3/server/storage/RecordStorageShould.java

```diff
@@ -194,6 +195,39 @@ public void write_record_bulk() {
         close(storage);
     }
 
+    @Test
+    public void rewrite_records_in_bulk() {
+        final RecordStorage<I> storage = getStorage();
+        final I id1 = newId();
```

Comment:
The point here was to show that some records are changed and some are not. I thought using a Collection here would make it more complicated since it would be less obvious what happens to the data.